components/Layout/Layout.tsx

```diff
@@ -19,6 +22,9 @@ export default function Layout({ children, pageTitle = 'LKS Kończyce Małe' })
 				<div className={styles.text}>{children}</div>
 				<div className={styles.shadowBox}></div>
 			</main>
+			<LatestNews />
+			<TableMatch />
+			<Sponsors />
```

Comment:
Those components should be only on the home page. The Layout component is kind of wrapper for each page and thats why we have here Header and Footer because we want those things in every page. This component needs to be refactored little bit because it has wrong usage now - we can discuss it in separate meeting.